### ISSUE TITLE
codap!

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1329]
Flannery!

I've tried to incorporate all of the feedback from Richmond. This included adding a bunch of CODAP starter files, trying to make workbook pages much clearer / more specific, and in general, overhauling lots of stuff.

I think what would be most valuable is for you to carefully read / work through the workbook pages to see if they feel effective to you. (Previously, you found them very confusing.) I've changed LOTS of things, so am also worried about weird inconsistencies - please feel free to go ahead and change any typos / errors if you see them.

I'm hoping you can review as far as codap-transformers2!

Let me know if you have questions. Can't wait to get more amazing feedback from you, so glad you're back.

R

**Comment by flannery-denny** [comment]
CODAP exploration sticking points

- [ ] add a row (this was tricky for me... I'd suggest offering a hint... I just needed to start typing and then hit return
- [ ] delete a row (hint: this is called a case in the menu)
- [ ] delete a column on the table (this is called an attribute - you use that word further down - suggest connecting the words right away)
- [ ] closing the table and reopening it lost me the column that I had deleted... even though I never hit save... why? how would I go about getting it back without going back to the link from the workbook page?
- [ ] guessing not all students will know what populated v unpopulated attributes are.
